test/Microsoft.Identity.Abstractions.Tests/Microsoft.Identity.Abstractions.Tests.csproj

```diff
@@ -4,7 +4,7 @@
     <TargetFrameworks>net7.0;net462</TargetFrameworks>
     <Nullable>enable</Nullable>
     <IsPackable>false</IsPackable>
-	<LangVersion>9.0</LangVersion>
+	<LangVersion>11</LangVersion>
```

Comment:
we should add net8.0 instead of 7 for the tests

Reply:
I was able to make the appropriate adjustments!